src/main/java/seedu/duke/command/TagCommand.java

```diff
@@ -13,40 +16,54 @@
 public class TagCommand extends Command {
 
     public static final String COMMAND_WORD = "tag";
-    private static final String ADD_TAG_MESSAGE = "Added the tag to the note!";
-    private static final String REMOVE_TAG_MESSAGE = "Removed the tag from the note!";
+    private static final String ADD_TAG_MESSAGE = "Added the tag to the note! ";
+    private static final String REMOVE_TAG_MESSAGE = "Removed the tag from the note! ";
 
     private static final String COMMAND_USAGE = COMMAND_WORD + ": Tags or untags a note. Parameters: "
             + PREFIX_DELIMITER + PREFIX_INDEX + " INDEX "
             + PREFIX_DELIMITER + PREFIX_TAG + " TAG_NAME [TAG_COLOR]";
 
-    private int index;
-    private String tagName;
-    private String tagColor;
+    private final int index;
```

Comment:
Private final right, so should be all CAPS? Can just commit directly from my suggestion :D
```suggestion
    private final int INDEX;
```

Reply:
The intellji gave warning saying that the field may be final so I changed it. Will be reverting that since the int cannot be final. Likewise for the rest of the similar cases below